proto/viam/app/data/v1/data.proto

```diff
@@ -58,6 +58,9 @@ service DataService {
   // BoundingBoxLabelsByFilter gets all string labels for bounding boxes from data based on given filter.
   rpc BoundingBoxLabelsByFilter(BoundingBoxLabelsByFilterRequest) returns (BoundingBoxLabelsByFilterResponse);
 
+  // UpdateBoundingBox updates the bounding box with the given image and bounding box ID.
```

Comment:
Maybe UpdateBoundingBox updates the bounding box associated with a given binary ID and bounding box ID?